### PR TITLE
Ember Data notes 01 november 2017

### PR DESCRIPTION
**Pull request by hjdivad**

### CODE REVIEW

File: ember-data/2017-11/november-01.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# ember-data Meeting Notes - 2017-11-01
+
+## Attendees
+
+## Agenda
```
Comment (hjdivad) [inline ember-data/2017-11/november-01.md]: Reply here with agenda items

**Comment by hjdivad** [comment]
This is the format I'd suggested in this week's meeting @pangratz @bmac @runspired @dgeb

File: ember-data/2017-11/november-01.md (hunk 1 above)
Comment (runspired) [reply ember-data/2017-11/november-01.md]: Follow up on solving various use cases unload via addon

Comment (hjdivad) [reply ember-data/2017-11/november-01.md]: in particular
 - addon for transitive-unload
 - addon for soft-delete

Comment (rwjblue) [reply ember-data/2017-11/november-01.md]: Discuss testing API's:

* https://github.com/emberjs/data/issues/5247 - `store.createRecord` should be async?
* Meta issue around creating new-style `setupSerializerTest(hooks)` (as pseudo coded in [my blog post](http://rwjblue.com/2017/10/23/ember-qunit-simplication/#why))

Comment (runspired) [reply ember-data/2017-11/november-01.md]: Discuss https://github.com/emberjs/data/pull/5230 and whether/how to support multi-table inheritance.

Comment (runspired) [reply ember-data/2017-11/november-01.md]: Potentially discuss allowing unique indexes against simple (non-compound) attributes

**Comment by workmanw** [comment]
👋 -- Obviously I'm not a team member and don't have a say in what is/isn't discussed, but I've read these for years to keep up to date on what's happening in ember, ember-cli, and ember-data. 

I'd really like to see the outcome of a discussion around all the unloadRecord issues that began with 2.12. I'm one of at least 20 people who are still stuck on older versions and it'd be really great to at least know these things are being discussed, if not actively worked on.

**Comment by runspired** [comment]
The outcome is roughly that we are going to investigate enabling the `soft-delete` and `purge` use-cases via addon.  We're stuck in a hard place because we can't go back as that would reintroduce the bugs this change fixed, but nor can we fix forward as the issue is that there are three fundamentally different use-cases involved in how folks were using unloadRecord for better or for worse.

**Comment by workmanw** [comment]
I'd be interested to hear more, but this is probably not the right venue to ask these questions.

**Comment by locks** [comment]
@workmanw https://discuss.emberjs.com/ 😉

**Comment by erichonkanen** [comment]
hello, curious if there was/is any discussion on the [natural-key issue for models?](https://github.com/emberjs/data/issues/5237)

**Comment by runspired** [comment]
@erichonkanen no, but I believe this will fix that: https://github.com/emberjs/data/pull/5249

**Comment by hjdivad** [comment]
cc @bmac @pangratz @runspired @dgeb @igort @rwjblue

**Pull request merged**